### ISSUE TITLE
Jetbrains Rider not showing on editors list

### ISSUE THREAD

**Issue by ctigrisht** [FlaxEngine/FlaxEngine#1386]
Since Jetbrains has changed their install directories via toolbox, rider is not detected as an editor anymore.
Please fix the runtime check.

**Comment by eLeSTRaGo-Dev** [comment]
For Linux platform in script `Source/Editor/Scripting/CodeEditors/RiderCodeEditor.cpp` add new search path.

```
void RiderCodeEditor::FindEditors(Array<CodeEditor*>* output)
{
    ...
    SearchDirectory(&installations, localAppDataPath / TEXT(".local/share/JetBrains/Toolbox/apps/rider/"));
    ...
}
```

**Comment by mafiesto4** [comment]
Merged #1390 and #1391 + backported them to `master` branch.

**Comment by van800** [comment]
https://github.com/FlaxEngine/FlaxEngine/pull/1390#issuecomment-1724960495
